app/main.py

```diff
@@ -1,3 +1,12 @@
 def cache(func):
-    # Write your code here
-    pass
+    lis = []
```

Comment:
Bad and non-informative variable name.

It is better to use the dictionary to store since the key will store the parameters, and the value is - the result of the execute function)